src/model/style.js

```diff
@@ -1,6 +1,9 @@
 import Color from 'color';
 import Cytoscape from 'cytoscape'; // eslint-disable-line
 
+export const NODE_SELECTOR = 'node';
+export const EDGE_SELECTOR = 'edge';
+
```

Comment:
Not sure if this change is necessary.  I just did it because I thought it would make sense to use the same constant instead of using the string 'node' and 'edge' everywhere.

Reply:
Makes sense